samples/nrf9160/modem_shell/src/drivers/nrf91_non_offloaded_dev.c

```diff
@@ -0,0 +1,507 @@
+/*
```

Comment:
Couldn't this be just a regular driver, available to everyone, instead of part of the sample?

Reply:
Maybe it could, at least at some point, eventually. To get it there, IMO opinion it would need still some development:
- support for multiple contexts where each are having different IP (not trivial)
- PDN lib could have: support for multiple event handlers and at least some of the utils functions of this file could be in pdn lib.

Reply:
You mean multiple handlers for the default PDP context?

Reply:
@lemrey I meant for PDN library events. Yes, for now the need is for default ctx, but eventually for all contexts.